b2b-sample/guardio-insurance-sample-application/components/settingsComponents/settings.js

```diff
@@ -41,20 +41,19 @@ export default function Settings(props) {
 
     const [activeKeySideNav, setActiveKeySideNav] = useState('1');
 
-    const signOutOnClick = () => orgSignout();
-
     const mainPanelComponenet = (activeKey, session) => {
         switch (activeKey) {
             case '1':
+
```

Comment:
are these new lines required?

Reply:
Fixed with https://github.com/wso2/samples-is/pull/416/commits/130d957728499e5e0e731eba0b663086f931fe95